### PR TITLE
Add zabbix_regexp_info module

### PR DESCRIPTION
**Pull request by masa-orca**
##### SUMMARY

Add zabbix_regexp_info module

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME

community.zabbix.zabbix_regexp_info
##### ADDITIONAL INFORMATION

I create info module which outputs value for community.zabbix.zabbix_regexp (not raw value). 

```paste below

```

### CODE REVIEW

**Review by pyrodie18 — APPROVED** [review]

**Comment by masa-orca** [comment]
@pyrodie18 this module will be added 3.3.0?

**Comment by pyrodie18** [comment]
Can add it whenever.  I approved it was gonna let you merge it

**Comment by masa-orca** [comment]
Sure.
If I merge, We need to release 2.3.0 after 2.2.0 without minor release. Is it OK to us?

File: tests/integration/targets/test_zabbix_regexp_info/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+- name: assert that regexp was NOT updated
+  ansible.builtin.assert:
+    that:
+      - zbxregexp_update.changed is sameas False
+
+- name: test - try to retrieve regexp
+  community.zabbix.zabbix_regexp_info:
+    name: File systems for discovery
+  register: zbxregexp_get
```
Comment (BGmot) [inline tests/integration/targets/test_zabbix_regexp_info/tasks/main.yml:19]: You don't use zbxregexp_get anywhere, should you create an "assert" for this?

**Review by BGmot — APPROVED** [review]
Thanks!

**Pull request merged**